### PR TITLE
Fix/fix minor issues

### PR DESCRIPTION
**Pull request by Vietprole**
- Trim the name input before validation in **checkFormValidity()**
- Fix wrong function names' format (Get, Put, Post, Delete -> get, put, post, del).
- Create variables for regex in **helpers.js**
- Apply CSS style for table row even/odd in **dashboard.css**.
- Remove styles that are directly on elements, replace by style on classes (line 17, 21 in main.js).

### CODE REVIEW

File: .gitattributes
Comment (hoangle-agilityio) [inline .gitattributes:1]: Please add it to git ignore. We don't need to push it on git

File: src/js/templates/delete-confirmation-modal.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ function closeDeleteConfirmationModal() {
 async function removeCustomer(event) {
   const { target } = event;
   target.disabled = true;
-  await Delete(`${API_BASE_URL}/${state.currentCustomer.id}`);
+  await del(`${API_BASE_URL}/${state.currentCustomer.id}`);
```
Comment (hoangle-agilityio) [inline src/js/templates/delete-confirmation-modal.js]: should be `delete` instead of

File: src/js/utils/helpers.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
 import sortingStates from '../constants/sort';
+// Regular expression variables
+const numberRegex = /\d/; // Check for numbers
```
Comment (hoangle-agilityio) [inline src/js/utils/helpers.js]: Please create `regex.js` in constants and create `object` to contains it. Ex
```
REGEX = {
   POSITIVE_VALID: ....
}
```

File: src/js/templates/delete-confirmation-modal.js (hunk 1 above)
Comment (Vietprole) [reply src/js/templates/delete-confirmation-modal.js]: "delete" is a reserved word by JS so I've changed those to getData, postData, putData, deleteData

File: src/js/utils/helpers.js (hunk 2 above)
Comment (Vietprole) [reply src/js/utils/helpers.js]: Ok, I've made them into constants.

File: .gitattributes
Comment (Vietprole) [reply .gitattributes:1]: Ok, I've add it to .gitignore.

**Pull request merged**